js/apps/admin-ui/src/clients/authorization/policy/PolicyDetails.tsx

```diff
@@ -101,6 +104,11 @@ export default function PolicyDetails() {
     ({ policy, policies }) => {
       reset({ ...policy, policies });
       setPolicy(policy);
+      if (policy !== undefined) {
+        if (policy.name === "Default Policy") {
+          setIsDisabled(true);
+        }
+      }
```

Comment:
Since this is the only place that `setIsDisabled` is set and the state is derived from the value of `policy` this doesn't actually have to be a state. Instead the value could be derived from the value of policy:

```ts
const [policy, setPolicy] = useState<PolicyRepresentation>();
const isDisabled = policy?.name === "Default Policy";
```